projects/media-viewer/src/lib/directives/drag-scroll.directive.ts

```diff
@@ -0,0 +1,49 @@
+import { Directive, ElementRef, HostListener, Input } from '@angular/core';
```

Comment:
I don't think we should have a separate folder for directives, they should just go where they're used. so if this is used with viewers it should just go there. that fits better with angular's domain driven approach.

Reply:
the folder decision was because, i realised we are starting to have multiple directives so i thought ill create a folder